rtl/eclrtl/eclregex.cpp

```diff
@@ -489,9 +490,15 @@ class CCompiledStrRegExpr : implements ICompiledStrRegExpr
         }
     }
 
+    void replaceTimed(ISectionTimer * timer, size32_t & outlen, char * & out, size32_t slen, char const * str, size32_t rlen, char const * rstr) const
+    {
+        SectionTimerBlock sectionTimer(timer);
+        replace(outlen, out, slen, str, rlen, rstr);
```

Comment:
trivial:  You could add a final qualifier to the class definition, which would allow the compiler to make this a non-virtual function call.
Same for other classes.